modules/apps/29-fee/ibc_middleware_test.go

```diff
@@ -635,6 +636,21 @@ func (suite *FeeTestSuite) TestOnAcknowledgementPacket() {
 			},
 			true,
 		},
+		{
+			"fail on no distribution by escrow account out of balance",
+			func() {
+				packetID := channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, suite.chainA.SenderAccount.GetSequence())
+				err := suite.chainA.GetSimApp().BankKeeper.SendCoinsFromModuleToAccount(suite.chainA.GetContext(), types.ModuleName, suite.chainA.SenderAccount.GetAddress(), smallAmount)
+				suite.Require().NoError(err)
+
+				// expectedBalance and expectedRelayerBalance don't change
+				expectedBalance = originalBalance.Add(smallAmount...)
+				expectedRelayerBalance = sdk.NewCoins()
+
+				suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), packetID, types.NewPacketFees([]types.PacketFee{packetFee}))
```

Comment:
on this test case, we should be checking that
- fee module is locked
- packet fees didn't get deleted

on other test cases, we should be checking that:
- fee module isn't locked
- packet fees did get deleted

Reply:
I will open a pr to this pr, since these are testing improvements we should of previously had